libs/libfido2/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2020 Linos Giannopoulos
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libfido2
+PKG_VERSION:=1.6.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/Yubico/libfido2/archive/
+PKG_HASH:=6aed47aafd22be49c38f9281fb88ccd08c98678d9b8c39cdc87d1bb3ea2c63e4
+
+PKG_FORTIFY_SOURCE:=0
```

Comment:
Don't bother with this.

Reply:
I've ran into issues with this one, because [libfido2](https://github.com/Yubico/libfido2/blob/7032a63c1e0eaa76e57d1eb7a5c85cfb65d446f3/CMakeLists.txt#L245) itself sets the -DFORTIFY_SOURCE=2, then OpenWRT does as well, if I understand correctly. If you have any suggestions on how to handle this, please let me know

Reply:
The aforementioned error thrown is about redefining the same macro: `Error: 'warning: "_FORTIFY_SOURCE" redefined'`

Reply:
sounds like -Wno-error should fix that?

Reply:
Unfortunately it doesn't. I've just tried it and it still fails when I remove `PKG_FORTIFY_SOURCE:=0`